src/mlagility/api/script_api.py

```diff
@@ -146,8 +148,27 @@ def benchmark_script(
     models_found: Dict[str, ModelInfo] = {}
 
     for script in input_scripts:
+        script_path, targets, encoded_input = decode_input_script(script)
+
+        # Filter scripts by label
+        if labels != []:
+            skip_script = False
+            user_labels = lab.to_dict(labels)
+            file_labels = lab.load_from_file(encoded_input)
+            for key in user_labels:
+                # Skip benchmarking if user_label is not in file_labels
+                if key not in file_labels:
+                    skip_script = True
+                    break
+                # A label key may point to multiple label values
+                # Skip if not all values of user_labels[key] are in file_labels[key]
+                elif not all(elem in user_labels[key] for elem in file_labels[key]):
+                    skip_script = True
+                    break
+            if skip_script:
+                continue
```

Comment:
```suggestion
            user_labels = lab.to_dict(labels)
            file_labels = lab.load_from_file(encoded_input)
            for key in user_labels:
                # Skip benchmarking if user_label is not in file_labels, or...
                # A label key may point to multiple label values
                # Skip if not all values of user_labels[key] are in file_labels[key]
                if key not in file_labels or not all(elem in user_labels[key] for elem in file_labels[key]):
                    continue
```

Can be a lot shorter 😸 

There's no need to `break` to help get to a `continue` statement, just `continue`.